src/vcacheoptimizer.cpp

```diff
@@ -195,10 +195,6 @@ void meshopt_optimizeVertexCacheTable(unsigned int* destination, const unsigned
 	TriangleAdjacency adjacency = {};
 	buildTriangleAdjacency(adjacency, indices, index_count, vertex_count, allocator);
 
-	// live triangle counts
```

Comment:
I'd prefer to keep the variable but alias it with adjacency.counts (i.e. `unsigned int* live_triangles = adjacency.counts;`)

Reply:
(the two uses below can keep using `live_triangles` then)

Reply:
Sure, will patch shortly.